roster2/tool/src/webapp/js/roster.js

```diff
@@ -22,7 +22,7 @@ roster.setupPrintButton = function () {
           Promise.all(Array.from(document.querySelectorAll("sakai-user-photo"))
             .map(sup => sup.updateComplete)).then(() => {
 
-            imagesLoaded("#roster-members-content", () => {
+            $().imagesLoaded("#roster-members-content", () => {
```

Comment:
is this more legible and common?

```
$('#roster-members-content').imagesLoaded(function() {
    // callback code
});
```

Reply:
@ottenhoff - I don't have a problem reverting to the more overtly jquery style. (I presume that current/recent instances of this code have attempted to move away from jquery.)

Reply:
isn't this your change that is moving from vanilla JS to jquery? I figured that was very purposeful. what is `$()` doing? I figured it was a jquery selector for everything

Reply:
The change is to get it to work. Your suggestion also seems to work (barring tests to be implemented for another jira... see comments for the corresponding jira). Absent changes to this line, console errors are thrown... cannot find imagesLoaded.

Reply:
@adrianfish are you okay with moving from vanilla JS back to jquery here? If not, do you have any help here?